etc/taskcluster/macos/states/generic-worker.sls

```diff
@@ -44,12 +44,27 @@
         clientId: {{ pillar["client_id"] }}
         accessToken: {{ pillar["access_token"] }}
         livelogSecret: {{ pillar["livelog_secret"] }}
+    - watch_in:
+      - service: net.generic.worker
 
 {{ bin }}/generic-worker new-openpgp-keypair --file {{ home }}/key:
   cmd.run:
     - creates: {{ home }}/key
     - runas: worker
 
+{{ home }}/run:
+  file.managed:
+    - mode: 744
+    - user: {{ user }}
+    - template: jinja
+    - contents: |-
+        #!/bin/sh
+        # generic-worker overwrites its config file to fill in defaults,
+        # but we want to avoid touching config.json here
```

Comment:
try to split this into its own file if possible

Reply:
Removed as part of another change.